rewriter/Struct.cc

```diff
@@ -112,13 +112,30 @@ vector<ast::ExpressionPtr> Struct::run(core::MutableContext ctx, ast::Assign *as
         }
     }
 
+    UnorderedSet<core::NameRef> seenNames;
     for (auto &arg : send->posArgs()) {
-        auto sym = ast::cast_tree<ast::Literal>(arg);
-        if (!sym || !sym->isSymbol()) {
+        auto lit = ast::cast_tree<ast::Literal>(arg);
+        if (!lit) {
             return empty;
         }
-        core::NameRef name = sym->asSymbol();
-        auto symLoc = sym->loc;
+
+        core::NameRef name;
+        if (lit->isSymbol()) {
+            name = lit->asSymbol();
+        } else if (lit->isString()) {
+            name = lit->asString();
+        } else {
+            return empty;
+        }
+
+        auto [it, inserted] = seenNames.insert(name);
+        if (!inserted) {
+            if (auto e = ctx.beginError(lit->loc, core::errors::Rewriter::InvalidStructMember)) {
+                e.setHeader("Duplicate member '{}' in Struct definition", name.show(ctx));
```

Comment:
```suggestion
                e.setHeader("Duplicate member `{}` in `{}` definition", name.show(ctx), "Struct");
```

Using the pattern

```
`{}`
```

will mean that that substring gets highlighted cyan in terminal output.